### PR TITLE
Fix issue with 'Linux' and 'Windows' notation in Korean docs

### LINKED ISSUE #22360: Issue with 'Linux' and 'Windows' notation in Korean docs
**This is a Bug Report**

**Problem:**
In some translated docs into Korean, 'Linux' and 'Windows' were not translated.

**Proposed Solution:**
Replace 'Linux' and 'Windows' with '리눅스' and '윈도우' in the following pages:
1. 'Linux':
- concepts/configuration/organize-cluster-access-kubeconfig.md
- concepts/services-networking/service.md
- concepts/workloads/pods/pod.md
- concepts/workloads/pods/init-containers.md
- concepts/policy/pod-security-policy.md
- concepts/cluster-administration/networking.md
- reference/glossary/docker.md
- setup/production-environment/tools/kops.md
- setup/production-environment/container-runtimes.md
- setup/learning-environment/minikube.md
- tasks/tools/install-minikube.md
- tasks/tools/install-kubectl.md
- tasks/access-application-cluster/configure-access-multiple-clusters.md

2. 'Windows':
- concepts/configuration/organize-cluster-access-kubeconfig.md
- concepts/services-networking/service.md
- setup/production-environment/windows/user-guide-windows-containers.md
- setup/learning-environment/minikube.md
- tasks/tools/install-minikube.md
- tasks/tools/install-kubectl.md
- tasks/administer-cluster/kubeadm/adding-windows-nodes.md
- tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md
- tasks/access-application-cluster/configure-access-multiple-clusters.md

**Page to Update:**
https://kubernetes.io/...

/language ko

### PR DESCRIPTION
**Pull request by pjhwa**
Closes #22360 

This PR replaces 'Linux' and 'Windows' with '리눅스' and '윈도우' in the following pages:

1. 'Linux':
- concepts/configuration/organize-cluster-access-kubeconfig.md
- concepts/services-networking/service.md
- concepts/workloads/pods/pod.md
- concepts/workloads/pods/init-containers.md
- concepts/policy/pod-security-policy.md
- concepts/cluster-administration/networking.md
- reference/glossary/docker.md
- setup/production-environment/tools/kops.md
- setup/production-environment/container-runtimes.md
- setup/learning-environment/minikube.md
- tasks/tools/install-minikube.md
- tasks/tools/install-kubectl.md
- tasks/access-application-cluster/configure-access-multiple-clusters.md

2. 'Windows':
- concepts/configuration/organize-cluster-access-kubeconfig.md
- concepts/services-networking/service.md
- setup/production-environment/windows/user-guide-windows-containers.md
- setup/learning-environment/minikube.md
- tasks/tools/install-minikube.md
- tasks/tools/install-kubectl.md
- tasks/administer-cluster/kubeadm/adding-windows-nodes.md
- tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md
- tasks/access-application-cluster/configure-access-multiple-clusters.md

### CODE REVIEW

**Comment by gochist** [comment]
@pjhwa 감사합니다!
/kind bug
/lgtm

**Comment by gochist** [comment]
저자과 제가 같은 회사 소속이므로 되도록이면 다른 승인자께서 승인해주셨으면 합니다.
/unassign
/assign @seokho-son

**Comment by seokho-son** [comment]
감사합니다.. ^^
/approve

File: content/ko/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md [hunk 1]
```diff
@@ -9,16 +9,16 @@ weight: 30
...
 
 
 
 ## {{% heading "prerequisites" %}}
  {{< version-check >}}
 
-* Windows 컨테이너를 호스팅하는 Windows 노드를 구성하려면
-[Windows Server 2019 라이선스](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing) 이상이 필요하다.
+* 윈도우 컨테이너를 호스팅하는 윈도우 노드를 구성하려면
+[윈도우 서버 2019 라이선스](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing) 이상이 필요하다.
```
Comment (sftim) [inline content/ko/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:21]: nit: The product name is Windows Server 2019, according to the page linked to.

Comment (pjhwa) [reply content/ko/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:21]: @sftim I agree. And I think it should be link to Korean page. I'll update on the next branch.

**Pull request merged**